summit/roiGeneration.m

```diff
@@ -286,11 +286,11 @@
 
 
 % Determine number of wells per row
-wells_per_row = round((x_new_upper_right - x_new_upper_left) / horzspacing);
+wells_per_row = round((x_new_upper_right - x_new_upper_left) / horzspacing) + 1;
 struct.wells_per_row = wells_per_row;
 
 % Determine number of rows
-rows = round((y_new_lower_right - y_new_upper_right) / vertspacing);
+rows = round((y_new_lower_right - y_new_upper_right) / vertspacing) + 1;

```

Comment:
I added this factor of 1 because we were leaving out the last row/column without it. Can you please check to make sure I did this correctly.